qutebrowser/browser/webelem.py

```diff
@@ -42,7 +42,7 @@
 SELECTORS = {
     Group.all: ('a, area, textarea, select, input:not([type=hidden]), button, '
                 'frame, iframe, link, [onclick], [onmousedown], [role=link], '
-                '[role=option], [role=button], img'),
+                '[role=option], [role=button], img, [ng-click], [ngClick], [data-ng-click], [x-ng-click]'),
```

Comment:
Can you put the Angular JS selectors in a separate line and add a comment like `# Angular 1` or so above it? Otherwise it's not really clear what they're here for. I think having a test for one of them is enough, so you won't need to change anything there.

Reply:
Fixed.

Reply:
BTW, you saw I changed ``links`` to ``all`` in test?

Reply:
Yep, and it's probably fine. I need to look at it again in detail, but don't worry about it for now.